theme/static/js/hs_file_browser.js

```diff
@@ -430,11 +443,22 @@ function showFileTypeMetadata(){
          var json_response = JSON.parse(result);
          $("#fileTypeMetaDataTab").html(json_response.metadata);
          $(".file-browser-container, #fb-files-container").css("cursor", "auto");
+         $("#btn-add-keyword-filetype").click(onAddKeywordFileType);
```

Comment:
OK, way down here I understand the onAddKeyword* functions. That's too long a wait :)